src/modules/mc_pos_control/mc_pos_control_main.cpp

```diff
@@ -331,6 +335,9 @@ MulticopterPositionControl::MulticopterPositionControl() :
 	_global_vel_sp_pub(nullptr),
 	_manual_thr_min(this, "MANTHR_MIN"),
 	_manual_thr_max(this, "MANTHR_MAX"),
+	_vel_x_deriv(this, "VELD"),
+	_vel_y_deriv(this, "VELD"),
```

Comment:
This is adding 3 low pass filters. One for each velocity component. It is using the same parameter names for each of the low pass filter blocks.